### PR TITLE
Update light pattern

### PR DESCRIPTION
**Pull request by rzhai2**

### CODE REVIEW

File: lightstrip.py [hunk 1]
```diff
@@ -87,28 +90,20 @@ def assign_mode(self, current_mode: str, last_mode: str):
...
         elif current_mode == modes["NO_CAMS_WORKING_CUSTOM"]:
-            self.primary == colors["NO_COLOR"]
-            self.pattern_function = patterns.static
-
-        elif current_mode == modes["DISABLED_WITH_DEFAULT_AUTO"]:
-            self.primary = colors["HOT_PINK"]
-            self.pattern_function = patterns.static
-
-        elif current_mode == modes["DISABLED_WITH_CUSTOM_AUTO"]:
             self.primary = colors["BLUE"]
```
Comment (aschokking) [inline lightstrip.py:105]: are there ways to have the colors make more semantic sense? like no cameras working should maybe be angry red? Some yellow, all green?

File: patterns.py [hunk 2]
```diff
@@ -19,25 +19,7 @@ def breathing(light):
     for pixel in range(light.PIXEL_COUNT):
         light.neopixel[pixel] = tuple(x * (multiplier/2) for x in light.primary)
     light.neopixel.show()
-        
 
-# Alternates between two color of lights, Ex.
-# Loop 1: All green
-# Loop 2: All red
-def alternating(light):
```
Comment (aschokking) [inline patterns.py]: Why delete this vs leaving it as an option?

**Review by aschokking — APPROVED** [review]
Looks reasonable

File: lightstrip.py (hunk 1 above)
Comment (rzhai2) [reply lightstrip.py:105]: I was afraid that then we can’t tell whether it is default auto or not.

**Pull request merged**